test/verifiedTests/base/testIO/testConvertCobraLP2mps.m

```diff
@@ -0,0 +1,30 @@
+% The COBRAToolbox: testConvertCobraLP2mps.m
+%
+% Purpose:
+%     - test the convertCobraLP2mps function
+%
+% Authors:
+%     - Jacek Wachowiak
+global CBTDIR
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testConvertCobraLP2mps'));
+cd(fileDir);
+
+% test variables
+LPProblem = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'ecoli_core_model.mat']);
+LPProblem_2 = rmfield(LPProblem, 'csense');
+LPProblem_2 = rmfield(LPProblem_2, 'osense');
+
+% function outputs
+OK = convertCobraLP2mps(LPProblem);
+OK_2 = convertCobraLP2mps(LPProblem_2);
+
+% test
+assert(isequal(OK, 1));
```

Comment:
This test only tests the output of the function, but it doesn't check whether the output is actually correct. Can you read the MPS file and then compare the models? A similar test is already here: https://github.com/opencobra/cobratoolbox/blob/master/test/verifiedTests/base/testMPS/testMPS.m

Reply:
Ok, I will follow it